Dockerfile

```diff
@@ -1,8 +1,17 @@
 FROM alpine:3.11
 
+
+RUN echo 'https://repository.walmart.com/content/repositories/alpine-v38/community' > /etc/apk/repositories \
```

Comment:
How is this related to finalizers?

Reply:
It isn't, I need to revert that it was to get around company proxies